test/server/test_responses.py

```diff
@@ -45,7 +45,7 @@ def test_settings_page_requires_auth():
     user = ScreenshotsClient()
 
     resp = requests.get(urljoin(user.backend, "/settings"))
-    assert resp.status_code == 403
+    assert resp.status_code == 200
```

Comment:
We should change this test to confirm that the response was redirected (otherwise the test no longer really tests anything, i.e., doesn't test authentication logic). This is a [response object](http://docs.python-requests.org/en/master/api/#requests.Response) and an be checked for `resp.is_redirect` or probably better would be to check `resp.url == user.backend + "/"`